index.js

```diff
@@ -25,3 +25,45 @@ mongoose
   .catch(error => {
     console.error('Error connecting to the database', error);
   });
+
+  Recipe.create({
+    title: "Asian Glazed Chicken Thighs - version one",
+    level: "Amateur Chef",
+    ingredients: [
+      "1/2 cup rice vinegar",
+      "5 tablespoons honey",
+      "1/3 cup soy sauce (such as Silver Swan®)",
+      "1/4 cup Asian (toasted) sesame oil",
+      "3 tablespoons Asian chili garlic sauce",
+      "3 tablespoons minced garlic",
+      "salt to taste",
+      "8 skinless, boneless chicken thighs"
+    ],
+    cuisine: "Asian",
+    dishType: "main_course",
+    image: "https://images.media-allrecipes.com/userphotos/720x405/815964.jpg",
+    duration: 40,
+    creator: "Chef LePapu"
+  })
+  .then(recipeDoc => console.log(`The title is: ${recipeDoc.title} and this is the whole recipe:${recipeDoc}`))
+  .catch(err =>
+    console.log(`Creating a new recipe went wrong! Try again 😞 ${err}`)
+  );
+
+Recipe.insertMany(data)
+.then(recipeDocs => console.log(`new recipes inserted: ${recipeDocs}`))
+.catch(err =>
+  console.log(`Inserting new recipes went wrong! Try again 😞 ${err}`)
+);
+
+Recipe.updateOne({title:"Rigatoni alla Genovese"},{$set: {duration:100}})
```

Comment:
Your findOneAndUpdate already knows that the second key is the one to update once it recognises it, so your $set is unnecessary here ($set usually adds new fields to a document; it can also update, but it is used more with the method .aggregate())